docs/get-started/lotus/send-and-receive-fil.md

```diff
@@ -44,16 +44,43 @@ Remember that your will only see the latest balance when your daemon is fully sy
 
 ## Sending FIL
 
-Sending some FIL can be achieved by running:
+Sending some FIL from default wallet can be achieved by running:
 
 ```bash
 lotus wallet send <address> <amount>
```

Comment:
```suggestion
lotus wallet send <target address> <amount>
```
I used "target" like in line 67, but @hsanjuan may have an opinion on wording. Either way, the goal is to clarify that it's the address you're sending _to_.